packages/p2p-v2/src/pages/orders/screens/Orders/OrdersEmpty/OrdersEmpty.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { BASE_URL } from '@/constants';
+import { DerivLightOrderIcon } from '@deriv/quill-icons';
+import { ActionScreen, Button, Text, useDevice } from '@deriv-com/ui';
+
+const OrdersEmpty = () => {
+    const { isMobile } = useDevice();
+    const textSize = isMobile ? 'lg' : 'md';
+    const history = useHistory();
+    return (
+        <div className={isMobile ? 'py-16 px-[1.6rem]' : ''}>
```

Comment:
on desktop does this need padding? cause we can do it like this instead since you're manipulating the x and y values of it only for mobile

```suggestion
        <div className='lg:p-0 py-16 px-[1.6rem]'>
```

Reply:
lg values depends on the configuration for tailwind right? as of now i think it would be better to stick to our configuration until we set a config for tailwind or use one from deriv-com/ui.. and we will also be supporting tablets.. so styles need to be applied for mobile

Reply:
hmmm true, but there are other places where we've been using lg in other places in our components, so i think its ok to stick to it for the time being, since we want to make it follow current prod